views/js/pciCreator/dev/mathEntryInteraction/runtime/mathEntryInteraction.js

```diff
@@ -541,20 +549,29 @@ define([
                     equal:  { label: '=',           latex: '=',         fn: 'write',    desc: 'Equal'},
                     lower:  { label: '<',           latex: '<',         fn: 'write',    desc: 'Lower than'},
                     greater: { label: '>',          latex: '>',         fn: 'write',    desc: 'Greater than'},
-                    subscript: { label: 'x&#8345;', latex: '_',         fn: 'cmd',      desc: 'Subscript'},
-                    lbrace: { label: '{',           latex: '{',   fn: 'cmd',      desc: 'Left brace/curly bracket'},
-                    rbrace: { label: '}',           latex: '}',  fn: 'cmd',      desc: 'Right brace/curly bracket'},
-                    lparen: { label: '(',           latex: '(',   fn: 'cmd',      desc: 'Left parenthese/round bracket'},
-                    rparen: { label: ')',           latex: ')',  fn: 'cmd',      desc: 'Right parenthese/round bracket'},
+                    subscript: { label: '_',        latex: '_',         fn: 'cmd',      desc: 'Subscript'},
```

Comment:
The new subscript doesn't have an `x` before it. What do you think about `x&#8336;`?

Reply:
I like it! Awesome!